meinberlin/apps/projects/templates/meinberlin_projects/project_detail.html

```diff
@@ -61,7 +61,7 @@ <h1 class="title">{{ project.name }}</h1>
                             {% if project.image_copyright %}
```

Comment:
@hom3mad3 that was added at the request of @CarolingerSeilchenspringer when we made the copyright required in the dashboard to encourage the initiators to add it to old projects, no new projects will have this displayed as an image cannot be added without copyright

Reply:
but now i'm confused about the double quotes again?! will look up the rules in the dslint rules

Reply:
@hom3mad3 the copyright is now required when adding images, the "copyright missing" string was added for old projects that were created before we made image copyrights required

Reply:
yep that's it, i'm not sure why we couldn't add a default in the database for the old projects when we changed it to required, was a backend thing

The quotes thing i'm not sure about, it's a django template thing so maybe @m4ra is a better person to ask if we should keep the quote rules, I guess code consistency is good to have. 
I checked and it should be double quotes so this PR is linted correctly (i guess maybe because the linting was merged in between the commits and I had make lint correcting after doing a rebase). I do not know why they both passed linting when I checked?!

Reply:
> @philli-m design also has a `Source:` before the `project.image_copyright }}`

@hom3mad3 Do you mean 'Image:' ? I don't see a design that adds 'Source:' [design link](https://app.zeplin.io/project/5b051eef7aaa417d2227c55c/screen/651d856a095b522d99f968fb) 
Also as this PR is for the tabs so I wouldn't change the hero as that's covered in a separate PR